### PR TITLE
[CVP-2063] cvp cluster pool claim for 4.6 & 4.8

### PR DESCRIPTION
**Pull request by sonam1412**

### CODE REVIEW

File: clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-east-1_clusterpool.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  imageSetRef:
+    name: ocp-4.6.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: cvp-aws-credentials
+      region: us-east-1
```
Comment (dirgim) [inline clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-east-1_clusterpool.yaml:24]: I believe we will need to use a different region for each cluster pool. I propose the us-west-1 region for v4.6.

File: clusters/hive/pools/cvp/cvp-ocp-4-8-amd64-aws-us-east-1_clusterpool.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  imageSetRef:
+    name: ocp-4.8.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: cvp-aws-credentials
+      region: us-east-1
```
Comment (dirgim) [inline clusters/hive/pools/cvp/cvp-ocp-4-8-amd64-aws-us-east-1_clusterpool.yaml:24]: I believe we will need to use a different region for each cluster pool. I propose the us-east-2 region for v4.6.

File: clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-east-1_clusterpool.yaml (hunk 1 above)
Comment (sonam1412) [reply clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-east-1_clusterpool.yaml]: ack

File: clusters/hive/pools/cvp/cvp-ocp-4-8-amd64-aws-us-east-1_clusterpool.yaml (hunk 2 above)
Comment (sonam1412) [reply clusters/hive/pools/cvp/cvp-ocp-4-8-amd64-aws-us-east-1_clusterpool.yaml]: ack

**Comment by jsztuka** [comment]
Im not seeing changed files as they were here previously for some reason.

**Review by jsztuka — APPROVED** [review]
/LGTM

**Pull request merged**